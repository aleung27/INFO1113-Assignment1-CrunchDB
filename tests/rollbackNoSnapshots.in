SET g 8
ROLLBACK 1
ROLLBACK 0
SNAPSHOT
DROP 1
ROLLBACK 1
LIST ENTRIES
