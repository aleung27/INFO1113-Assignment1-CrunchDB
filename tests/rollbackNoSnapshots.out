> ok

> no such snapshot

> no such snapshot

> saved as snapshot 1

> ok

> no such snapshot

> g [8]

